tests/xilinx/latches.ys

```diff
@@ -0,0 +1,19 @@
+read_verilog latches.v
+design -save read
+
+proc
+async2sync # converts latches to a 'sync' variant clocked by a 'super'-clock
+flatten
+synth_xilinx
+equiv_opt -assert -map +/xilinx/cells_sim.v synth_xilinx # equivalency check
+design -load postopt # load the post-opt design (otherwise equiv_opt loads the pre-opt design)
+
+design -load read
+
+synth_xilinx
+flatten
+cd top
+select -assert-count 1 t:LUT1
+select -assert-count 2 t:LUT3
+select -assert-count 3 t:$_DLATCH_P_
```

Comment:
Given this is obviously broken (`$_DLATCH_P_` is a Yosys cell that Vivado won't recognise), is this really a useful test?

Reply:
I checked this case in Xilinx Vivado and it synthesize LDCE cells, but Yosis don't. I think, this test is useful and will be updated when LDCE will be supported. Or I'm not right?

Reply:
The concern is someone seeing this test might think it is correct behaviour to have `$_DLATCH_P_` cells at the end of synthesis. Perhaps you could add a comment and/or create an issue?

Reply:
OK. Will add a comment for expected behavior. And a new issue.

Reply:
An issue for LDCE cell support:
[](https://github.com/YosysHQ/yosys/issues/1387)

Reply:
Please update test with: https://github.com/YosysHQ/yosys/pull/1396/commits/8c2dd518b908644c9181526e193b8211734e8cef

Reply:
Resolved in #1420